frontend/app/views/spree/products/_filters_desktop.html.erb

```diff
@@ -83,6 +83,14 @@
                 </div>
               <% end %>
             <% end %>
+
+            <%= form_with url: permitted_params, id: 'filterPriceRange', method: :get do |f| %>
```

Comment:
@damianlegawiec Using `permitted_params` on the form url builds a correct url, but it doesn't send it on submitting the form - since it's not present as a form field.

It works for `POST` form, but then other filters won't work with that.

So maybe we should go back to using `setNewUrl`? What do you think?

Reply:
Alternatively we can not use a form and just have 2 inputs and a button link with data. It'd be similar to the `DONE` button on the mobile filters view.